### PR TITLE
Version 3.0.0 w/dependency on @okgrow/auto-analytics npm package

### PR DESCRIPTION
**Pull request by ghost**
I think this is about ready to go. What's happened?

1. Removed all code that is now handled by the @okgrow/auto-analytics npm package.
2. Added dependency to the @okgrow/auto-analytics npm package.
3. Cleaned up and streamlined remaining code.
4. Updated examples.
5. Updated README.

Eventually this will become an npm package too, but since it depends on the Meteor accounts package(s). It seems to make sense to leave this as an Atmosphere package for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,33 @@
...
+## Ad-blocker
+When running your Meteor app in "development mode" ad-blocking web-browser extensions may block the `okgrow:analytics` package due to the word "analytics" in the package name. This only occurs when running Meteor in "development mode" because files are not bundled together and minified. To work around this issue you can disable your ad-blocker when running in development mode.
 
-To test that your application runs whilst an ad-blocker is enabled you can run your Meteor app with the following command:
+To test that application with an ad-blocker, run your Meteor app in production mode with this command:
 
 `meteor run --production --settings settings.json`
 
-**NOTE:** If an Adblocker is enabled the expected behaviour is that your analytic events will not be received. You will see an error message in your console reporting the events being blocked.
+**NOTE** If an ad-blocker is enabled the expected behavior is that analytic events will not be received. You'll see an error message in your console reporting the events being blocked.
```
Comment (cfnelson) [inline README.md:30]: nit: picking on our english `behaviour` 😝  😂

**Comment by cfnelson** [comment]
@ccuilla Seems like there are conflicts ?

**Review by cfnelson — CHANGES_REQUESTED** [review]
The branch seems to be 17 commits behind master. Need to merge master into this branch before merging. Then it should look good to go. 👍

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "analytics",
-  "version": "2.1.3",
+  "version": "3.0.0",
   "description": "Complete automatic pageview tracking with Google Analytics, Mixpanel, KISSmetrics (and more) integration for JavaScript applications.",
```
Comment (cfnelson) [inline package.json]: Should this be changed to mention this part of the package is Meteor specific?

File: package.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Package.describe({
   name: 'okgrow:analytics',
-  version: '2.1.3',
+  version: '3.0.0',
   summary: 'Complete Google Analytics, Mixpanel, KISSmetrics (and more) integration for Meteor',
```
Comment (cfnelson) [inline package.js]: Same comment about the package.json, should this mention it is Meteor specific?

**Review by cfnelson — APPROVED** [review]

**Review by cfnelson — APPROVED** [review]
Liking the summary description 👍

**Review by cfnelson — DISMISSED** [review]
@ccuilla Changes look good but have been unable to start/run the examples in this pr. What steps did you take to run the examples? 

The example for setting the `document.title` for `iron-router` does not work for first page load & also when you use the back navigation button. I have not yet tested the `flow-router` or `react-router` example.

**Comment by ghost** [comment]
I think this guy is ready to be published.

**Comment by cfnelson** [comment]
👍

**Review by cfnelson — APPROVED** [review]

**Pull request merged**